File: Java/2416.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {

    public static void main(String[] args) throws IOException {
    
        InputStreamReader ir = new InputStreamReader(System.in);
        BufferedReader in = new BufferedReader(ir);
        String[] line = in.readLine().trim().split(" ");
        int a = Integer.parseInt(line[0]);
        int b = Integer.parseInt(line[1]);
        
        System.out.println((a % b));
        
    }
}

File: C#/2416.cs
using System; 

class URI {

    static void Main(string[] args) { 

        string[] line = Console.ReadLine().Trim().Split(' ');
        int a = int.Parse(line[0]);
        int b = int.Parse(line[1]);
        int r = a % b;
        Console.WriteLine(r);

    }

}